src/irr/asset/COBJMeshFileLoader.h

```diff
@@ -170,24 +170,29 @@ class COBJMeshFileLoader : public asset::IAssetLoader
             SObjMtl() : Bumpiness (1.0f), Illumination(0),
```

Comment:
all of this should move to a `CPipelineLoaderMTL` and `CDescriptorSetLoaderMTL` (both read the same file, produce different assets)